src/modules/callouts/pages/CalloutAdminOverviewPage.vue

```diff
@@ -110,15 +111,32 @@ import {
 import ActionButton from '../components/ActionButton.vue';
 import CalloutSummary from '../../../components/CalloutSummary.vue';
 import AppAlert from '../../../components/AppAlert.vue';
+import { updateCallout, createCallout } from '../../../utils/api/callout';
 
 const props = defineProps<{
   callout: GetMoreCalloutData;
 }>();
 const { t } = useI18n();
 
 const route = useRoute();
+const router = useRouter();
 const wasJustCreated = route.query.created !== undefined;
 const wasJustUpdated = route.query.updated !== undefined;
+const wasJustReplicated = route.query.replicated !== undefined;
 
 const calloutLink = computed(() => `/callouts/${props.callout.slug}`);
+
+async function replicateThisCallout() {
+  const newCalloutData = {
+    ...props.callout,
+    slug: props.callout.slug + '-copy',
+    title: props.callout.title + ' copy',
+    status: undefined,
```

Comment:
Changes look good :+1:. Final request is to set the start and expiry date to null to make the new callout a draft:
```ts
{
  ...,
  starts: null,
  expires: null
};
```